src/js/media-time-display.js

```diff
@@ -86,7 +87,7 @@ class MediaTimeDisplay extends MediaTextDisplay {
   #slot;
 
   static get observedAttributes() {
-    return [...super.observedAttributes, ...CombinedAttributes, 'disabled'];
+    return [...super.observedAttributes, ...CombinedAttributes, 'disabled', 'notoggle'];
```

Comment:
nitpick(blocking): add `NO_TOGGLE: 'notoggle'` to `Attributes` object instead of declaring it inline here.